DFC.ServiceTaxonomy.GraphSync/Extensions/MergeNodeExtensions.cs

```diff
@@ -62,6 +62,24 @@ public static class MergeNodeExtensions
         {
             List<T>? values;
             JArray? jarray = (JArray?)content[contentPropertyName];
+
+            if (jarray != null)
```

Comment:
Looks like this has changed the behaviour of this method.
If jarray is null, then it now returns an empty list, whereas before it returned null (while still returning null if JArrayType == JTokenType.Null, and behaving differently to the AddArrayProperty overload below).
We should return null if jarray is null, which would be best achieved by not checking for != null here and just returning the result of AddArrayProperty direct.